core/src/main/scala/kafka/security/auth/SimpleAclAuthorizer.scala

```diff
@@ -227,6 +227,10 @@ class SimpleAclAuthorizer extends Authorizer with Logging {
     aclCache.toMap
   }
 
+  override def getSupportedPrincipalTypes(): Set[String] = {
+    Set(KafkaPrincipal.USER_TYPE)
```

Comment:
Why are you overriding this method since the implementation is the same as in the interface?